gnucash/gnome/gnc-plugin-page-register.cpp

```diff
@@ -3898,29 +3898,31 @@ gnc_plugin_page_register_cmd_reverse_transaction (GSimpleAction *simple,
     {
         const char *rev = _("A reversing entry has already been created for this transaction.");
         const char *jump = _("Jump to the transaction?");
-        if (gnc_verify_dialog (GTK_WINDOW (window), TRUE, "%s\n\n%s", rev, jump))
-            goto jump_to_trans;
```

Comment:
A better way to accomplish this would be to extract into new functions the two blocks, say `maybe_jump_to_existing` and `maybe_create_reverse_transaction`, then
```
if (auto new_trans{xaccTransactionGetReversedBy(trans)
{
    if (!maybe_jump_to_existing(/*…*/))
    {
        LEAVE ("reverse already exists");
        return;
    }
}
else
{
     if (!maybe_create_reverse_transaction(/*…*/))
     {
          LEAVE ("reverse cancelled");
          return;
      }
}
```